cmd/virt-handler/virt-handler.go

```diff
@@ -150,6 +153,28 @@ func main() {
 	ws := new(restful.WebService)
 	ws.Route(ws.GET("/api/v1/namespaces/{namespace}/vms/{name}/console").To(console.Console))
 	restful.DefaultContainer.Add(ws)
+
+	labelSelector, err := labels.Parse("daemon in (virt-handler)")
+	if err != nil {
+		fmt.Printf("Can't parse label selector: %s\n", err.Error())
+		return
+	}
+	pods, err := coreClient.CoreV1().Pods(k8sv1.NamespaceDefault).List(meta_v1.ListOptions{LabelSelector: labelSelector.String()})
+	if err != nil {
+		fmt.Printf("Can't find virt-handler pod: %s\n", err.Error())
+		return
+	}
+
+	virtHandlerUrl := pods.Items[0].GetSelfLink()
+	fmt.Println(virtHandlerUrl)
+	_, err = restClient.
```

Comment:
I really don't like the idea of a pod having permissions to modify its own pod definition.  In the future we're going to need to begin locking down every component so they only have access to exactly what they need to operate. Virt-handler shouldn't need to be able to modify anything related to the KubeVirt deployment.

If virt-handler has permissions to modify anything, it might be vm definitions, but I'm not convinced that will be necessary either.